nodewatcher/modules/monitor/sources/snmp/models.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+
+from django.apps import apps
+from django.contrib.postgres import fields as postgres_fields
+from django.db import models
+from django.utils.translation import ugettext_lazy as _, ugettext
+
+from polymorphic import models as polymorphic_models
+
+from nodewatcher.core import validators as core_validators
+from nodewatcher.core.registry import fields as registry_fields, registration
+from nodewatcher.core.generator.cgm import models as cgm_models
+
+
+class SNMPDaemonConfig(cgm_models.PackageConfig):
+    """
+    SNMP Daemon configuration.
+    """
+
+    community = models.CharField(max_length=100)
+    sysLocation = models.CharField(max_length=100)
+    sysContact = models.CharField(max_length=100)
```

Comment:
Please rename this to `sys_contact` (e.g., no camel case in variable names). Also, any specific reason why this is not just `contact`? Also, can this be inferred from somewhere else (for example we actually have the node's contact already available as the maintainer, so perhaps that information can be reused and this field is not needed at all)?